src/components/Header.tsx

```diff
@@ -62,10 +63,13 @@ export default function Header() {
     [targets, guesses]
   );
   const numGuesses = guesses.length;
-  const practice = useSelector((s) => s.game.practice);
-  const title = practice
-    ? `Practice Duotrigordle`
-    : `Daily Duotrigordle #${id}`;
+  const mode = useSelector((s) => s.game.mode);
+  const title =
```

Comment:
This feels like it should be a switch statement but I couldn't work out the syntax.